src/pages/database/overview/DBOverview.jsx

```diff
@@ -213,7 +221,7 @@ const mapDispatchToProps = (dispatch, ownProps, state) => {
       );
     },
     handleSelection: collectionname => {
-      dispatch(set('uiState.database.selectedCollection', collectionname));
+      dispatch(set(`uiState.database.${selectedDb}.selectedCollection`, collectionname));
```

Comment:
We use use camelCase for all variable names. Thus `collectionname` should be `collectionName`